files/default/sql/ddl/2.6.0__initial_tables.sql

```diff
@@ -1788,6 +1788,17 @@ CREATE TABLE IF NOT EXISTS `feature_store_kafka_connector` (
     ) REFERENCES `hops`.`hdfs_inodes` (`parent_id`, `name`, `partition_id`) ON DELETE CASCADE ON UPDATE NO ACTION
 ) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
 
+
+CREATE TABLE feature_store_gcs_connector (
```

Comment:
We use single quotes on all entity names, e.g. database name, table name, columns:
```suggestion
CREATE TABLE `feature_store_gcs_connector` (
```